src/darwin/Framework/CHIP/CHIPDevicePairingDelegateBridge.h

```diff
@@ -40,8 +40,8 @@ class CHIPDevicePairingDelegateBridge : public chip::Controller::DevicePairingDe
     void OnCommissioningComplete(chip::NodeId deviceId, CHIP_ERROR error) override;
 
 private:
-    id<CHIPDevicePairingDelegate> mDelegate;
-    dispatch_queue_t mQueue;
+    _Nullable id<CHIPDevicePairingDelegate> mDelegate;
+    _Nullable dispatch_queue_t mQueue;
```

Comment:
Does using `nullable` instead of `_Nullable` here work?  If so, would be somewhat better.

Reply:
Compiler tells me nullable is an unknown keyword. Placing seems to be very picky as well.

I actually do not know what syntaxes are supported, ended up using something that seems to be used in other places and works.